CMakeLists.txt

```diff
@@ -49,6 +49,15 @@ endif ()
 
 set (CMAKE_CXX_STANDARD 11)
 
+# Use C++11
+if (CMAKE_VERSION VERSION_LESS "3.1")
+    if (CMAKE_CXX_COMPILER_ID STREQUAL "GNU")
+        set (CMAKE_CXX_FLAGS "--std=gnu++11 ${CMAKE_CXX_FLAGS}")
```

Comment:
Any reason not to use c++14? I've had it set locally ever since I've started working on this project and everything seems to work just fine.